src/common/cockpitwebserver.c

```diff
@@ -1085,8 +1093,13 @@ on_accept_certificate (GTlsConnection *conn,
                        GTlsCertificateFlags errors,
                        gpointer user_data)
 {
-  /* Only used during testing */
-  g_assert (cockpit_webserver_want_certificate == TRUE);
+  /* Only accept unknown CAs, e. g. for passing on the certificate to sssd */
+  if (errors != G_TLS_CERTIFICATE_UNKNOWN_CA) {
```

Comment:
Where is the CA for the certificate verified? Is it included in the chain? Does sssd verify the whole chain? Or is that one of the reasons that "unsafe".

Reply:
I don't feel like cockpit-ws is in a position to even know the admin/IdM policy to be able to do more than just checking the validity. The intention is to leave that to sssd -- in other words, if the admin said "I trust that this certificate identifies the user", then we should abide that. But I added a TODO item (to the description check list for now) to verify that with the stakeholder.

Reply:
Security team confirmed that cockpit-ws should not do certificate validation, and leave this to sssd/IdM.